uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/list/ListHabitsMenu.kt

```diff
@@ -122,31 +124,40 @@ class ListHabitsMenu @Inject constructor(
             }
 
             R.id.actionSortColor -> {
-                behavior.onSortByColor()
+                { behavior.onSortByColor() }.delay(SORT_DELAY_MILLIS)
                 return true
             }
```

Comment:
We don't need to delay the menu actions; changing the order should be done immediately. Could you please remove all calls to `delay` in this file?